lib/cog/adapters/slack/rtm_connector.ex

```diff
@@ -108,7 +108,10 @@ defmodule Cog.Adapters.Slack.RTMConnector do
                               fn() -> handle_command(room, user_id, text, state) end}
                          end
     unless message == nil do
-      Slack.API.send_message(channel, message)
+      {:ok, room} = Slack.API.lookup_room(id: channel)
+      room = %{"id" => room.id,
+               "name" => room.name}
+      Slack.API.send_message(room, message)
```

Comment:
Actually it'd be really nice if `send_message` just took the room map, but I think in the executor we deserialize json into a map with string keys so that's why we need to do that weird conversion on line 112.